app/i18n/es/sub.php

```diff
@@ -228,6 +228,16 @@
 		'title' => 'Título',
 		'title_add' => 'Añadir fuente RSS',
 		'ttl' => 'No actualizar de forma automática con una frecuencia mayor a',
+		'unicityCriteria' => array(
+			'_' => 'Article unicity criteria',	// TODO
+			'forced' => '<span title="Block the unicity criteria, even when the feed has duplicate articles">forced</span>',	// TODO
+			'help' => 'Relevant for invalid feeds.<br />⚠️ Changing the policy will create duplicates.',	// TODO
+			'id' => 'Standard ID (default)',	// TODO
```

Comment:
I need some understanding for `Standard ID`:
Is it the GUID (Global Unique ID)?

Reply:
Yes, except that it is not a GUID but a unique ID within a given feed, and varies depending on the version of RSS /Atom.

See:
* RSS 2.0 `guid` https://www.rssboard.org/rss-specification#ltguidgtSubelementOfLtitemgt
* Atom `id` https://www.rfc-editor.org/rfc/rfc4287#page-19
* RSS 1.0 item URI https://web.resource.org/rss/1.0/spec#s5.5

Reply:
Would `ID (default)` also be a good String?
It is difficult to translate `Standard` and `default` in the same String because both could be translated to something like `Standard ID (Standardeinstellung)`. Too much `Standard` in 1 String :D

Reply:
Yes, I think this is fine enough